pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -10691,3 +10787,14 @@ spec:
 		})
 	}
 }
+
+func getContextBasedOnFeatureFlag(featureFlag string) context.Context {
+	featureFlags, _ := config.NewFeatureFlagsFromMap(map[string]string{
+		"enable-api-fields": featureFlag,
+	})
+	cfg := &config.Config{
+		FeatureFlags: featureFlags,
+	}
+
+	return config.ToContext(context.Background(), cfg)
+}
```

Comment:
instead of creating this new function, could you reuse `withEnabledAlphaFields`?

https://github.com/tektoncd/pipeline/blob/5188483f1d2d9fb34d0df910c1e84be30676f941/pkg/reconciler/pipelinerun/pipelinerun_test.go#L1493-L1497

Reply:
done